suite-common/wallet-core/src/fiat-rates/fiatRatesActions.ts

```diff
@@ -0,0 +1,67 @@
+import { createAction } from '@reduxjs/toolkit';
+
+import {
+    Account,
+    CoinFiatRates,
+    TickerId,
+    WalletAccountTransaction,
+} from '@suite-common/wallet-types';
+import { Network } from '@suite-common/wallet-config';
+
+import { actionPrefix } from './constants';
+
+type FiatRatesPayload = NonNullable<CoinFiatRates['current']>;
+type UpdateFiatRatePayload = {
+    ticker: TickerId;
+    payload: FiatRatesPayload;
+};
+type UpdateTransactionFiatRatePayload = Array<{
+    txid: string;
+    account: Account;
+    updateObject: Partial<WalletAccountTransaction>;
+    ts: number;
+}>;
+type LastWeekRatesPayload = {
+    symbol: Network['symbol'] | string;
+    tickers: NonNullable<CoinFiatRates['lastWeek']>['tickers'];
+    ts: number;
+};
+
+const removeFiatRate = createAction(
+    `${actionPrefix}/removeFiatRate`,
+    (payload: TickerId): { payload: TickerId } => ({
```

Comment:
it's not necessary to type return type in all these `createAction`.

`(payload: TickedId) => ({ payload })` will have correcy type inherited

Remove also for other actions